paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/commands/PaimonShowColumnsCommand.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.spark.commands
+
+import org.apache.paimon.spark.SparkTable
+import org.apache.paimon.spark.leafnode.PaimonLeafRunnableCommand
+import org.apache.paimon.table.FileStoreTable
+
+import org.apache.spark.sql.{Row, SparkSession}
+import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference, GenericRow}
+import org.apache.spark.sql.types.{Metadata, StringType}
+import org.apache.spark.unsafe.types.UTF8String
+
+case class PaimonShowColumnsCommand(v2Table: SparkTable)
+  extends PaimonLeafRunnableCommand
+  with WithFileStoreTable {
+  override def table: FileStoreTable = v2Table.getTable.asInstanceOf[FileStoreTable]
```

Comment:
maybe this command can also apply to the system table, not just `FileStoreTable`. if yes, modify here and add a UT for that.

Reply:
Thanks for comments，but I had a try but seems not find a way to compatible with system table and v2table due to their not use same interface. Maybe we can just support FileStoreTable firstly？